### DISCUSSION TITLE
Github Beta => self hosted runners on kubernetes

### DISCUSSION THREAD

**Discussion by GoranHalvarsson** [actions/actions-runner-controller#2444] [Questions] [answered]
Hello

I just saw a video from GitHub, they where talking about self hosted runners on K8. 
Is it similar to arc, or it's something else?
https://youtu.be/sAv88cjWpG4

**Comment by kevholmes** [comment, marked as answer]
I'm pretty sure ARC is now officially GitHub's chosen method for self-hosted runners on k8s. They are working on an entirely new scaling method but that's currently only supported for a select set of clients who have been asked to join the beta on github.com rather than GHES etc. I believe this is because the GH API that's supporting this new method is behind a feature flag of some kind and hasn't been released as GA.

**Reply by GoranHalvarsson** [reply]
Thanks :-) That is music to my ears. I'm a big fan of ARC - we are using it heavily in production.   🎉

**Reply by kevholmes** [reply]
Same!! Love this project 😄
